lib/net/http.rb

```diff
@@ -736,6 +737,20 @@ def set_debug_output(output)
     # it raises a Net::ReadTimeout exception. The default value is 60 seconds.
     attr_reader :read_timeout
 
+    # Maximum number of times to retry an idempotent request in case of
+    # Net::ReadTimeout, IOError, EOFError, Errno::ECONNRESET,
+    # Errno::ECONNABORTED, Errno::EPIPE, OpenSSL::SSL, Timeout::Error
+    # Should be non-negative integer number. Zero means no retries.
+    # The default value is 1.
+    def max_retries=(retries)
+      if retries.is_a?(Fixnum) || retries < 0
```

Comment:
I don't think we're supposed to reference `Fixnum` anymore.

https://www.ruby-lang.org/en/news/2016/12/25/ruby-2-4-0-released/